swift_scality_backend/diskfile.py

```diff
@@ -214,14 +225,14 @@ def app_iter_ranges(self, ranges, content_type, boundary, size):
 class DiskFile(object):
     """A simple sproxyd pass-through
 
```

Comment:
No docstring to describe the client_collection param. I know it's self explanatory but, it's always nice to have all param taken into account.